sw/device/examples/hello_usbdev/hello_usbdev.c

```diff
@@ -75,10 +75,23 @@ static void usb_receipt_callback_1(uint8_t c) {
   ++usb_chars_recved_total;
 }
 
+static void usb_send_str(char *s, usb_ss_ctx_t *ss) {
+  while (*s) usb_simpleserial_send_byte(ss, *s++);
+}
+
 static dif_gpio_t gpio;
 
+#define PINFLIP 0
+#define DIFFIO 0
```

Comment:
Do these need to be defines? If not, use a `static const kConst` instead.

Reply:
Rewoked this code to read the values from GPIO bits. These can be set with switches on the FPGA and through the GPIO dpi in verilator. Enabled the more automated testing of all four combinations with verilator and simplifies manual testing of the two supported on FPGA.